docs/proposals/timeseries-partitioning-in-compactor.md

```diff
@@ -264,4 +295,128 @@ T2 partition 4 - Hash(timeseries label) % 4 == 3 && % 8 == 7
 T3 partition 8 - Hash(timeseries label) % 8 == 7
 ```
 
+### Compaction Partitioning Examples
+
+#### Ideal case:
+
+All source blocks were previously compacted through partitioning compaction. In this case for each time interval, the number of blocks belong to same time interval would be 2^x if multiplier is set to 2.
+
+```
+Time intervals:
+T1, T2, T3
+
+Source blocks:
+T1: B1, B2
+T2: B3, B4, B5, B6
+T3: B7, B8, B9, B10, B11, B12, B13, B14
+
+Total indices size of all source blocks:
+200G
+```
+
+Partition Number = (200G / 64G) => round up to next 2^x = 4
+
+Partitioning:
+* For T1, there are only 2 blocks which is < 4. So
+
+    * B1 (index 0 in the time interval) can be grouped with other blocks having N % 4 == 0 or 2. Because 0 % 2 == 0.
+    * B2 (index 1 in the time interval) can be grouped with other blocks having N % 4 == 1 or 3. Because 1 % 2 == 1.
+
+* For T2,
+
+    * B3 (index 0 in the time interval) can be grouped with other blocks having N % 4 == 0.
+    * B4 (index 1 in the time interval) can be grouped with other blocks having N % 4 == 1.
+    * B5 (index 2 in the time interval) can be grouped with other blocks having N % 4 == 2.
+    * B6 (index 3 in the time interval) can be grouped with other blocks having N % 4 == 3.
+
+* For T3,
+
+    * B7 (index 0 in the time interval) can be grouped with other blocks having N % 4 == 0.
+    * B8 (index 1 in the time interval) can be grouped with other blocks having N % 4 == 1.
+    * B9 (index 2 in the time interval) can be grouped with other blocks having N % 4 == 2.
+    * B10 (index 3 in the time interval) can be grouped with other blocks having N % 4 == 3.
+    * B11 (index 4 in the time interval) can be grouped with other blocks having N % 4 == 0.
+    * B12 (index 5 in the time interval) can be grouped with other blocks having N % 4 == 1.
+    * B13 (index 6 in the time interval) can be grouped with other blocks having N % 4 == 2.
+    * B14 (index 7 in the time interval) can be grouped with other blocks having N % 4 == 3.
+
+Compaction Plans:
+
+* Partition ID: 0 \
+  Partition Number: 4 \
+  Blocks: B1, B3, B7, B11
+* Partition ID: 1 \
+  Partition Number: 4 \
+  Blocks: B2, B4, B8, B12
+* Partition ID: 2 \
+  Partition Number: 4 \
+  Blocks: B1, B5, B9, B13
+* Partition ID: 3 \
+  Partition Number: 4 \
+  Blocks: B2, B6, B10, B14
+
+---
+
+#### Only Level 1 Blocks:
```

Comment:
nit: when I read this title (and other titles) I was bit confused, so I had to read the description closely to understand what you are trying to communicate. I would suggest you to change the title to something like `Scenario: source blocks are all level 1 blocks`.